backend/src/main/java/casa/eric/erzieherapp/backend/security/SecurityConfig.java

```diff
@@ -0,0 +1,30 @@
+package casa.eric.erzieherapp.backend.security;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration @EnableWebSecurity
+public class SecurityConfig {
+
+    @Value("${app.url}")
+    private String appUrl;
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        http
+            .csrf(AbstractHttpConfigurer::disable)
```

Comment:
Diese Zeile wird dir in Sonar Probleme machen, da es eine Sicherheitslücke darstellt.

Klicke gleich einfach auf die Sonar Meldung die dir sagt das dein Security Rating schlecht ist.
Wenn du auf Sonar weitergeleitet wirst musst du dich einloggen und dann kannst du das Issue markieren (Nicht als resolved, sondern als "I know what im doing" oder so :D ) dann wirst du nicht mehr von Sonar blockiert und kannst den PR mergen